osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -75,6 +75,7 @@ public class BeatmapInfo : IEquatable<BeatmapInfo>, IJsonSerializable
 
         public bool LetterboxInBreaks { get; set; }
         public bool WidescreenStoryboard { get; set; }
+        public float StoryboardAspect => WidescreenStoryboard ? 16 / 9f : 4 / 3f;

```

Comment:
Can `Storyboard` receive a `BeatmapInfo` as a parameter and calculate this internally? I don't want to be polluting `BeatmapInfo` like this.